frb_codegen/src/ir/field.rs

```diff
@@ -64,4 +72,15 @@ impl IrField {
 
         format!("{enum_name}.{variant_name}")
     }
+
+    /// Takes the provided name and surrounds it with the `mirror_` prefix if the field is in a
+    /// mirrored enum.
+    pub fn try_name_mirror(&self, name: String) -> String {
+        let mirrored_name = self.ty.mirrored_nested();
+        if self.settings.is_in_mirrored_enum && mirrored_name.is_some() {
```

Comment:
Wait a minute, maybe we already find the answer: Is it possible we simply remove `is_in_mirrored_enum` flag, because is_in_mirrored_enum can be derived from `mirrored_name.is_some()`?

Reply:
I hoped so as well, but I did a test and sadly there were edge cases in which it was not. I also added a few more unit tests for this

Reply:
I will try to fix `Vec<Enums>` at one point, so maybe then I’ll find a better way for it